python/ray/train/constants.py

```diff
@@ -112,6 +112,9 @@ def _v2_migration_warnings_enabled() -> bool:
 # Defaults to 0
 RAY_TRAIN_ENABLE_STATE_TRACKING = "RAY_TRAIN_ENABLE_STATE_TRACKING"
 
+# Environment variable for Train execution callbacks
+RAY_TRAIN_CALLBACKS_ENV_VAR = "RAY_TRAIN_CALLBACKS"
```

Comment:
can you add this to the `v2/constants` file since it's only relevant there

Reply:
do we need to propagate this env variable from the driver for the Tune + Train use case?

Reply:
Hmm for Tune + Train we'd technically to do that for every Train environment variable, right? 

Might be more user friendly to just have practices around setting environment variables across the cluster.